### ISSUE TITLE
Question: What are the extra NPR features?

### ISSUE THREAD

**Issue by Ulf3000** [dillongoostudios/goo-blender#1]
sorry for asking here , but i couldnt find a single information on the web.

**Comment by sodiufas** [comment]
watch blender talk, mostly some nodes in shading, maybe something with greeace pencil

**Comment by Ulf3000** [comment]
yes ive watched it already 😁 , the edge shader for the highlighted edges is very awesome !
